pkg/model/resource/options.go

```diff
@@ -188,8 +241,14 @@ func (opts *Options) NewResource(c *config.Config, doResource bool) *Resource {
 
 	res.Package = pkg
 	res.Domain = opts.Group
-	if domain != "" {
+	if domain != "" && opts.Group != "" {
 		res.Domain += "." + domain
+	} else if opts.Group == "" && c.IsV3() {
```

Comment:
nit 
```suggestion
	} else if opts.Group == "" && !c.IsV2() {
```

Reply:
supported for any upper version that we might add as well.

Reply:
Done